custom-dependencies/README.md

```diff
@@ -0,0 +1,79 @@
+# Managing Custom Dependencies and Patches
+
+In order to provide better separation and management of custom dependencies and patches, we've introduced two new files:
+
+- `custom-dependencies/composer.json`: For managing custom dependencies
+- `composer.patches.json`: For managing Drupal patches
+
+## Why are we making these changes?
+
+These changes allow for better organization and separation of concerns within the project. By isolating custom dependencies and patches in separate files, it reduces the likelihood of merge conflicts when updating the upstream and provides a clearer structure for managing project-specific dependencies and patches.
+
+---
+
+## Implementing the changes
+
+### Custom Dependencies
+
+1. Locate the `custom-dependencies/composer.json` file in the project.
+2. Remove all custom dependencies out of the root `composer.json` file, and move them into `custom-dependencies/composer.json`.
+3. Add, update, or remove custom dependencies from the "require" section of the `custom-dependencies/composer.json` file as needed.
+
+### Drupal Patches
+
+1. Locate the `composer.patches.json` file in the project root.
+2. Remove all custom patches out of the root `composer.json` file, and move them into `composer.patches.json`.
+3. Add, update, or remove patches in the "patches" section of the `composer.patches.json` file as needed.
+4. As we may need to add patches periodically, we recommend that you place your patches at the beginning of this file.
```

Comment:
Not a dealbreaker - but I'd suggest maybe flipping the recommendation - general users will be inclined by habit to place their patches at the bottom of the file, so perhaps our team should place patches at the top.

Reply:
Changed.